src/plugins/console/public/application/models/sense_editor/sense_editor.test.js

```diff
@@ -345,6 +345,46 @@ describe('Editor', () => {
     })
   );
 
+  const singleLineRequestBodyComment = {
+    prefix: 'PUT index_1/type1/1',
+    data: ['{', '  "f": 1', '// "f": 2', '}'].join('\n'),
+  };
+
+  const multiLineRequestBodyComment = {
+    prefix: 'PUT index_1/type1/2',
+    data: ['{', '  "f": 1', '/* "f": { "t": 2 } */', '}'].join('\n'),
+  };
+
+  utilsTest(
+    'request-body comments - single line',
+    singleLineRequestBodyComment.prefix,
+    singleLineRequestBodyComment.data,
+    callWithEditorMethod('getRequest', (request, done) => {
+      const expected = {
+        method: 'PUT',
+        url: 'index_1/type1/1',
+        data: [singleLineRequestBodyComment.data],
```

Comment:
This is a dumb question because it makes it obvious that I don't understand how these tests work. 😅  But from a naive perspective, it looks like we're basically comparing the input of `singleLineRequestBodyComment.data` to the output of `singleLineRequestBodyComment.data`. What are we verifying by doing so?